index.bs

```diff
@@ -942,6 +944,17 @@ spec:css-syntax-3;
 
     1.  Let |p| be [=a new promise=].
 
+    1. Let |topLevelBrowsingContext| be |settings|' [=top-level browsing context=].
+
+    1. [=map/For each=] |optionKey| → <var ignore>optionValue</var> of |options|:
+
+        1. If |topLevelBrowsingContext|'s [=active credential types=] [=set/contains=] |optionKey|,
+            [=reject=] |p| with null and return |p|.
```

Comment:
Wasn't sure what we should do (probably not reject p with null?)

Reply:
Probably an error, see https://www.w3.org/2001/tag/doc/promises-guide#reasons-should-be-errors

`TypeError` seems to be the most generic, or else see https://webidl.spec.whatwg.org/#dfn-error-names-table